sbt/src/sbt-test/sbt-stryker4s/test-2.11/src/main/scala/example/Example.scala

```diff
@@ -0,0 +1,5 @@
+package example
+
+object Example {
+  def canDrink(age: Int): Boolean = age >= 18
```

Comment:
I think we should also try to ensure the project works with 2.11. I couldn't find any breaking changes between 2.11 and 2.12, but I think we can use the [JavaConversions](https://www.scala-lang.org/api/2.12.3/scala/collection/JavaConversions$.html) deprecated in 2.12 with a scalacOption to not compile with [deprecation errors](https://tpolecat.github.io/2017/04/25/scalac-flags.html) (`-deprecation` and `-Xfatal-warnings`, I think)